packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -179,6 +179,40 @@ The Progress indicator component displays a subtitle "x of y steps completed" by
 />
 ```
 
+## Adding level 2 steps
+
+If one of the steps in a multi-step form has a secondary task, use a separate page for the secondary task. Let users know where they are by using a level 2 step in the progress indicator. A level 2 step sits under the main step. There can only be a single level 2 step for each main step.
+
+```jsx live
+<ProgressIndicator
+	hideSubtitle
+	items={[
+		{ href: '#', label: 'Introduction', status: 'done' },
+		{
+			href: '#',
+			label: 'Organisations',
+			status: 'started',
+			levelTwoItem: { label: 'Add an organisation', href: '#' },
+		},
+		{ href: '#', label: 'Review and submit', status: 'blocked' },
+	]}
+/>
+```
+
+### An example of a level 2 step
+
+Let’s say that a user is completing a multi-step application form. They want to change their pre-filled details, so they select the link to change their details.
```

Comment:
You could use the YourGov employees list as an example. Saves you having to code up a scenario from scratch for the sake of screenshots.

Reply:
Nice idea! Could you please point me towards this list 🙏

Reply:
Go here:
https://design-system.agriculture.gov.au/pr-preview/pr-1602/yourgov/

Sign in (any email and password works)

Charlies fast food

Apply for a permit

Fill out all the details in task 1

First page on Task 2 has the employee list